Sources/MapboxMaps/Foundation/Extensions/NSNumber.swift

```diff
@@ -3,15 +3,22 @@ import CoreGraphics
 import CoreLocation
 
 // MARK: - NSNumber
-public extension NSNumber {
+extension NSNumber {
 
     /// Converts an `NSNumber` to a `CGFloat` value from its `Double` representation.
-    var CGFloat: CGFloat {
+    public var CGFloat: CGFloat {
         CoreGraphics.CGFloat(doubleValue)
     }
 
     /// Converts the `Float` value of an `NSNumber` to a `CLLocationDirection` representation.
-    var CLLocationDirection: CLLocationDirection {
+    public var CLLocationDirection: CLLocationDirection {
```

Comment:
For later: can we make these internal?

Reply:
Agreed - let's tackle with the other similar changes.